week1/main.go

```diff
@@ -0,0 +1,64 @@
+package main
```

Comment:
Can run `go fmt` to clean up the code (indent, spaces ...)
https://blog.golang.org/go-fmt-your-code